### PR TITLE
Fix error in performance comparison in llama2 blog and some formatting fixes

### PR DESCRIPTION
**Pull request by MaanavD**
### Description
An error in the published perf comparison was discovered. Corrected results are included in this PR

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,25 +1,24 @@
...
-npm create svelte@latest
-
-# create a new project in my-app
-npm create svelte@latest my-app
-```
-
+---
+nav_exclude: true
+---
 ## Developing
```
Comment (natke) [inline README.md:4]: We could rename this to CONTRIBUTING.md?

File: _config.yml [hunk 2]
```diff
@@ -35,3 +35,8 @@ gh_edit_repository: 'https://github.com/microsoft/onnxruntime' # the github URL
 gh_edit_branch: 'gh-pages' # the branch that your docs is served from
 # gh_edit_source: docs # the source that your files originate from
 gh_edit_view_mode: 'tree' # "tree" or "edit" if you want the user to jump into the editor immediately
+nav_external_links:
```
Comment (natke) [inline _config.yml:38]: What impact does this have?

Comment (MaanavD) [reply _config.yml:38]: It creates a new nav link in the blogs, linking to the GH docs: 
image

Comment (natke) [reply _config.yml:38]: I don't think we need this. The docs on ONNX Runtime are for contributors. There might be some clean up we need to do on GitHub to make this distinction clear (as the docs all used to be in GitHub only)

Comment (MaanavD) [reply _config.yml:38]: @faxu can you comment on this change?

Comment (parinitarahi) [reply _config.yml:38]: I think its ok to have it for easy access. While the goal would be to clean up the docs on GH, this help with discoverability.

**Review by parinitarahi — APPROVED** [review]
The changes for updated compile numbers for the llama blog, and other changes are approved.

File: src/routes/blogs/accelerating-llama-2/+page.svelte [hunk 3]
```diff
@@ -18,11 +18,39 @@
...
 	<h1 class="text-5xl pb-2">Accelerating LLaMA-2 Inference with ONNX Runtime</h1>
-	<p class="text-neutral">By: <a href="https://www.linkedin.com/in/kunal-v-16315b94" class="text-blue-500">Kunal Vaishnavi</a> and <a href="https://www.linkedin.com/in/parinitaparinita/" class="text-blue-500">Parinita Rahi</a> </p>
+	<p class="text-neutral">
+		By: <a href="https://www.linkedin.com/in/kunal-v-16315b94" class="text-blue-500"
+			>Kunal Vaishnavi</a
+		>
+		and
+		<a href="https://www.linkedin.com/in/parinitaparinita/" class="text-blue-500">Parinita Rahi</a>
+	</p>
 	<p class="text-neutral">14TH NOVEMBER, 2023</p>
```
Comment (prasanthpul) [inline src/routes/blogs/accelerating-llama-2/+page.svelte:54]: seems like we should add a note that the blog was updated on 11/22 with revised numbers

Comment (MaanavD) [reply src/routes/blogs/accelerating-llama-2/+page.svelte]: Updated.

**Review by prasanthpul — APPROVED** [review]

**Pull request merged**